subgraphs/aave-v3/protocols/aave-v3/src/entities/event.ts

```diff
@@ -232,6 +232,12 @@ export function createLiquidate(
     market,
     PositionSide.LENDER
   );
+  const borrowerPosition = getOrCreateUserPosition(
+    event,
+    userAccount,
+    getMarket(debtAsset),
+    PositionSide.BORROWER
+  );
```

Comment:
Not sure what to do with the borrower position during liquidation, right now there's no reference to the liquidation, it is just closed.

Reply:
Good point. Do you think it'd be helpful if we changed `Liquidate` entity to include both borrower position and lender position?

Reply:
I do think it would be helpful, but it would also potentially make queries more complicated. Still, might be something to think about for next schema update?

Reply:
yep, included it in here: https://github.com/messari/subgraphs/issues/389

Reply:
How would you make the change without adding this position then?

Reply:
I agree that it is probably best to include the second position under `Liquidate`, either in a separate field or make Liquidate.position an array?

Reply:
yeah, let's do a separate field as discussed in Discord. Will improve this in the next schema upgrade (post Mainnet): https://github.com/messari/subgraphs/issues/389.